ecms_base/features/custom/ecms_hotel/config/install/core.entity_form_display.node.hotel.default.yml

```diff
@@ -0,0 +1,160 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.hotel.field_hotel_address
+    - field.field.node.hotel.field_hotel_body
+    - field.field.node.hotel.field_hotel_booking_notes
+    - field.field.node.hotel.field_hotel_booking_url
+    - field.field.node.hotel.field_hotel_cover_image
+    - field.field.node.hotel.field_hotel_main_image
+    - field.field.node.hotel.field_hotel_more_info_url
+    - field.field.node.hotel.field_hotel_phone
+    - field.field.node.hotel.field_hotel_rate
+    - field.field.node.hotel.field_hotel_rate_amount
+    - node.type.hotel
+  module:
+    - address
+    - link
+    - media_library
+    - path
+    - telephone
+    - text
+id: node.hotel.default
+targetEntityType: node
+bundle: hotel
+mode: default
+content:
+  created:
+    type: datetime_timestamp
+    weight: 10
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  field_hotel_address:
+    weight: 122
+    settings: {  }
+    third_party_settings: {  }
+    type: address_default
+    region: content
+  field_hotel_body:
+    weight: 121
+    settings:
+      rows: 5
+      placeholder: ''
+    third_party_settings: {  }
+    type: string_textarea
+    region: content
+  field_hotel_booking_notes:
+    weight: 127
+    settings:
+      rows: 5
+      placeholder: ''
+    third_party_settings: {  }
+    type: text_textarea
+    region: content
+  field_hotel_booking_url:
+    weight: 126
+    settings:
+      placeholder_url: ''
+      placeholder_title: ''
+    third_party_settings: {  }
+    type: link_default
+    region: content
+  field_hotel_cover_image:
+    type: media_library_widget
+    weight: 129
+    settings:
+      media_types: {  }
+    third_party_settings: {  }
+    region: content
+  field_hotel_main_image:
+    type: media_library_widget
+    weight: 130
+    settings:
+      media_types: {  }
+    third_party_settings: {  }
+    region: content
+  field_hotel_more_info_url:
+    weight: 128
+    settings:
+      placeholder_url: ''
+      placeholder_title: ''
+    third_party_settings: {  }
+    type: link_default
+    region: content
+  field_hotel_phone:
+    weight: 123
+    settings:
+      placeholder: ''
+    third_party_settings: {  }
+    type: telephone_default
+    region: content
+  field_hotel_rate:
+    weight: 124
+    settings:
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: string_textfield
+    region: content
+  field_hotel_rate_amount:
+    weight: 125
+    settings:
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: string_textfield
+    region: content
+  moderation_state:
+    type: moderation_state_default
+    weight: 100
+    settings: {  }
+    region: content
+    third_party_settings: {  }
```

Comment:
One last moderation state that should be removed.

Reply:
I believe this is expected, since we do have content_moderation enabled by default with the base install. While it's not assigned to a workflow, the moderation setting was showing for most content types regardless, so I left it in.

Reply:
👍